setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = dockerfile
-version = 2.1.0
+version = 2.2.0
```

Comment:
generally you shouldn't bump the version number as part of the PR -- I'll handle making a release after merging this -- if you could rebase this out that would be great!

Reply:
Haha sorry, this is just a leftover from my attempts to get Python to pick up the latest Go library (culprit turned out to be a stale wheel file that somehow ended up in the root folder 😅)